gravitee-am-ui/package.json

```diff
@@ -36,6 +36,7 @@
     "@angular/platform-browser-dynamic": "17.2.3",
     "@angular/platform-server": "17.2.3",
     "@angular/router": "17.2.3",
+    "@asciidoctor/core": "3.0.4",
```

Comment:
added as requested by ui-component. i tried with npm too but it didn't change anything.